### DISCUSSION TITLE
PHP 8.2 Support

### DISCUSSION THREAD

**Discussion by gligorkot** [craftcms/docker#79] [Q&A]
Hi all,

We've started using PHP 8.2 in our development lately and want to bring our CraftCMS websites up to the latest language version out there. Is there any plan to bring in a php-fpm 8.2 docker image any time soon?

Please let us know if we can help in any way.

Thanks,
G

**Comment by martijnvanonz** [comment]
Hi,

Is there any progress on this topic?

**Comment by jessedobbelaere** [comment]
PHP 8.1 active support ends in 3 months; I opened a pull request for 8.2 🤞 https://github.com/craftcms/docker/pull/82
Even got lucky with the Github PR numbering! 🚀 

Edit: PHP 8.2 is merged 👍 feel free to close this discussion
